factcast-store/src/main/java/org/factcast/store/registry/transformation/cache/InMemTransformationCache.java

```diff
@@ -121,6 +121,17 @@ public synchronized void invalidateTransformationFor(String ns, String type) {
     }
   }
 
+  @Override
+  public synchronized void invalidateTransformationFor(UUID factId) {
```

Comment:
sync seems wrong here, in order to avoid CMEs, we'need to sync on the cache itself (being a sync-collection).
This bug already existed in the other invalidate method.

fixed it.